specs/gridspec.js

```diff
@@ -0,0 +1,66 @@
+var chai = require('chai');
+var expect = chai.expect;
+var Grid = require('../src/grid');
+
+describe('Grid', function() {
+
+  var grid;
+  beforeEach(function(done) {
+    grid = new Grid(Object);
```

Comment:
Why are you injecting Object in here rather than using a spy library like sinon?
If not create a variable called `cell` that is an object and then inject that in